### ISSUE TITLE
Jitsi meet web interface missing some button ARIA roles and related attributes

### ISSUE THREAD

**Issue by tyrylu** [jitsi/jitsi-meet#7653]
### Description:
On the welcome page, pre join screen and some more places, the buttons are just divs without any semantic markup - once more, something similar happened with the toolbox items. They should at least get a button role and a tabindex, but then you would want to add a key down handler as well. Or, probably better, can not you use regular buttons? To add to the complexity, the pre join action button has a dropdown, but if not looking really closely with a screen reader, you would not notice it.

### Steps to reproduce:
Try creating and joining a meeting with any screen reader - NVDA on Windows, Orca on Linux etc.
### Expected behavior:
The buttons will be reported and will behave as buttons, there will be a way how to access the button dropdown, if there is any.

### Actual behavior:
The screen readers report only that there is something clickable, for the dropdown arrow, the something even has no text.

### Server information:

- Jitsi Meet version: master
- Operating System: not relevant

### Client information:

- Browser / app version: not relevant in this case, anything that has accessibility support and can show the web interface
- Operating System: everything that has a screen reader good enough to browse the web, e. g. Windows, Linux and others

### Additional information:
Could you reduce the count of custom controls, please? That would fix many of the issues just by virtue of the change...
